### DISCUSSION TITLE
Dynamic activation and deactivation of specific camera detection from home assistant.

### DISCUSSION THREAD

**Discussion by Islautopia** [blakeblackshear/frigate#15155] [Ask A Question]
Hi. I don't know if this has been documented or answered before, but I haven't been able to find it.
 Is there a way to dynamically activate and deactivate the detection of specific cameras in frigate from home assistant? The idea is to avoid alerts when we are at home. The triggering of alarms is controlled by the state of the alarm in home-assistant but the review page is filled with lots of detections of the family who have no interest walking around the house or in the garden when the detection makes no sense. I like to take a look at the detections page to see if everything is working ok. Having a review page with full of uninteresting detections when we are at home does not help improve the detections base that really matter.

**Comment by NickM-27** [comment]
Yes, using the Frigate integration in home assistant. You can disable detect, there is no switch currently to disable review items only

**Reply by Islautopia** [reply]
Sorry, that was a stupid question. As you point out, in the integration there is a switch to stop the detection.
